packages/nutui-replace-icons/src/replace-icons.ts

```
@@ -0,0 +1,35 @@
+  const targetLibrary = options.targetLibrary
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        if (sourceLibrary.indexOf(path.node.source.value) > -1) {
+          // 替换包名称
+          path.node.source.value = targetLibrary
+          path.node.specifiers.forEach((specifier) => {
+            // 根据 iconMappings 进行替换
+            const iconMappings = options.iconMappings
+            if (iconMappings && iconMappings[specifier.imported.name]) {
+              specifier.imported.name = iconMappings[specifier.imported.name]
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)